interfaz_controller/interfaz_controller.py

```diff
@@ -1,21 +1,17 @@
+import numpy as np
 import xlwings as xw
 
 
-def main():
+def main(workbook: xw.Book = None):
     wb = xw.Book.caller()
     sheet = wb.sheets[0]
-    if sheet["A1"].value == "Hello xlwings!":
-        sheet["A1"].value = "Bye xlwings!"
-    else:
-        sheet["A1"].value = "Hello xlwings!"
 
-
-@xw.func
-def hello(name):
-    return f"Hello {name}!"
+    params = sheet['B4:B8'].options(np.array, transpose=True).value
```

Comment:
Awesome! Now, what happens if the user of the excel sheet wants to add a row at the top of the file? Will this code work? Probably not, that's why I suggest using named ranges within excel, and then call the name of that range within xlwings. In that way you will make sure that range maintains no matter what the user does.

Reply:
Done

Reply:
Sorry, it is not ok. When I add a row in excel the calculation throws an error which means that the range is not persisted. Try using the Name Manager inside excel to assign a name to the range. Don't do it within the code.

Reply:
Yes, you are right, now it was corrected